### PR TITLE
[env/burn-staging] Mobile chat size

### PR DESCRIPTION
**Pull request by goran-peoski-work**
Resolves:
- https://github.com/sparkletown/internal-sparkle-issues/issues/1051

Before: 
sparkle-chat-size-02

After: 
sparkle-chat-size-01

### CODE REVIEW

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx [hunk 1]
```diff
@@ -44,15 +44,19 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
...
+  const venueChatTabStyles = classNames({
+    "chat-sidebar__tab": true,
     "chat-sidebar__tab--selected": isVenueChat,
   });
 
-  const privateChatTabStyles = classNames("chat-sidebar__tab", {
+  const privateChatTabStyles = classNames({
+    "chat-sidebar__tab": true,
     "chat-sidebar__tab--selected": isPrivateChat,
   });
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.tsx:61]: NOTE: Simpler to diff/read  when `classNames`' arguments aren't mixed but only strings or only object is used.

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx [hunk 2]
```diff
@@ -76,11 +80,7 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
           ) : (
             <>
               <FontAwesomeIcon icon={faChevronLeft} size="sm" />
-              <FontAwesomeIcon
-                className="chat-sidebar__controller__second-icon"
-                icon={faCommentDots}
-                size="lg"
-              />
```
Comment (goran-peoski-work) [inline src/components/organisms/ChatSidebar/ChatSidebar.tsx:83]: Redundant class, `:nth-child(2)` serves the purpose in the SCSS

**Review by goran-peoski-work — COMMENTED** [review]
Initial PR comments

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx (hunk 1 above)
Comment (denisdimitrov) [reply src/components/organisms/ChatSidebar/ChatSidebar.tsx:61]: @mike-lvov @sunny-viktoryia For or against, changing the `classNames` syntax to this one.
Old one: 
```tsx
const privateChatTabStyles = classNames("chat-sidebar__tab", {
    "chat-sidebar__tab--selected": isPrivateChat,
});
```

New one:
```tsx
const privateChatTabStyles = classNames({
    "chat-sidebar__tab": true,
    "chat-sidebar__tab--selected": isPrivateChat,
  });
```

**Review by denisdimitrov — APPROVED** [review]
LGTM!

File: src/components/organisms/ChatSidebar/ChatSidebar.tsx (hunk 1 above)
Comment (sunny-viktoryia) [reply src/components/organisms/ChatSidebar/ChatSidebar.tsx:61]: ooops, sorry, haven't noticed the comment. Thanks for mentioning me! ❤️ 

I'd prefer the old syntax. For me, it's easier to find what class names are added no matter what. But in the new syntax style, you'll explicitly need to find if there is a class name key with `true` value. what do you think?

**Pull request merged**